storage/cache/subtree_cache.go

```diff
@@ -294,10 +281,10 @@ func (s *SubtreeCache) getNodeHash(id tree.NodeID, getSubtree GetSubtreeFunc) ([
 			}
 		}
 		if c.Prefix == nil {
-			panic(fmt.Errorf("getNodeHash nil prefix on %v for id %v with px %x", c, id.String(), subKey))
+			panic(fmt.Errorf("getNodeHash nil prefix on %v for id %+v with px %x", c, id, subID))
```

Comment:
Can we propagate an error instead of the panic? It's a long standing issue that would be good to fix.

Reply:
#2406